dendrite-config.yaml

```diff
@@ -253,6 +253,18 @@ media_api:
     height: 480
     method: scale
 
+# Configuration for experimental MSC's
+mscs:
+  # A list of enabled MSC's
+  # Currently valid values are:
+  # - msc2836         (Threading, see https://github.com/matrix-org/matrix-doc/pull/2836)
+  mscs: []
+  database:
+    connection_string: file:mscs.db
+    max_open_conns: 10
```

Comment:
Just as a note to look at, but we need to make sure that the various `max_open_conns` in the sample config don’t collectively exceed 100.